src/app/app.component.ts

```diff
@@ -19,6 +19,7 @@ import {
 } from '../../projects/swimlane/ngx-ui/src/public_api';
```

Comment:
This file has some unused imports that we can remove now that most of it's contents have been moved out.